cluster-api/CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# Cluster API Development Guide
+
+## Cloud Provider Dev Guide
+
+### Overview
+
+The Cluster API is a Kubernetes project to bring declarative, Kubernetes-style APIs to cluster creation, configuration, and management. It provides optional, additive functionality on top of core Kubernetes.
+
+This document is meant to help OSS contributors implement support for cloud providers.
```

Comment:
(nit)

This line says the doc is for cloud providers. The next line seems to imply it is for cloud or on-prem. Would be nice to be consistent about the scope of the doc.

Reply:
Done